lib/generate/collector/product.js

```diff
@@ -28,21 +28,22 @@ const product = async (
 
     const bundleName = bundleConfig.name;
 
-    logger.info(`Collecting modules for bundle "${bundleName}".`);
+    logger.info(`Collecting modules for bundle ${bundleName}`);
```

Comment:
Why remove the quotes around `bundleName`?

Reply:
Hello Fredden. Same as with the use of backticks previously, I had changed the quotes to backticks across the entire file for uniformity. As for the quotes around the variable, this was due to my uncertainty of whether the variable would display the name or value if enclosed in quotes. I believe I may have been confusing the rules in Python with JavaScript rules.